docs/develop/worker-performance.mdx

```diff
@@ -141,13 +110,24 @@ TasksAddRate - TasksDispatchRate
 - Positive values of `X` indicate the backlog is growing by about `X` Tasks per second.
 - Negative values of `X` indicate the backlog is shrinking by about `X` Tasks per second.
 
-:::info Metric Accuracy
-
 While individual `add` and `dispatch` rates may be inaccurate due to Eager and Sticky Task Queues, the `BacklogIncreaseRate` reliably reflects the rate at which the backlog is shrinking or growing for backlogs older than a few seconds.
 
-:::
+## Evaluate Task Queue performance {#evaluate-worker-loads}
+
+A [Task Queue](https://docs.temporal.io/workers#task-queue) is a lightweight, dynamically allocated queue.
+[Worker Entities](https://docs.temporal.io/workers#worker-entity) poll the queue for [Tasks](https://docs.temporal.io/workers#task).
+The Temporal Service dynamically creates different [Task Queue types](/workers#task-queue) including Activity Task Queues, Workflow Task Queues, and Nexus Task Queues.
+These Task Queue types route their Tasks to Workers for Task completion.
+
+With an accurate estimate of backlog Tasks, you can determine the optimal number of Workers to deploy.
+Balance your Worker count with the number of Tasks to achieve the best performance.
+This approach minimizes Task backlog saturation and reduces idle Workers.
+
+Task Queue data provide numerical insights into your Task Queue activity and backlog characteristics.
+Use these numbers to tune your production deployments.
+Evaluate your Worker loads and assess whether you need to scale up or reduce your Worker deployment.
 
-### Fetch Worker metrics {#retrieve-worker-info}
+### Query Task Queue info with Temporal CLI {#cli-task-queue-info}
```

Comment:
This is nice. I would like to add another section similar to this but which says "Query Task Queue info with the Go SDK" and have a little code snippet. Does that work? I can share the code snippet here.

Reply:
RPS limits added, paralleled  in Go section, during our sync